honeybee_vtk/helper.py

```diff
@@ -0,0 +1,147 @@
+"""Functions to help in geometry translation."""
+
+from ladybug_geometry.geometry3d import Face3D, Point3D
+
+
+def get_mesh_points(boundary_points, holes_points=None):
+    """Convert a Face3D with holes in triangulated mesh and get vertices.
+
+    This function constructs a Ladybug Face3D object from vertices and creates
+    triangulated meshes from the generated Face3D object. The function then returns
+    the vertices of these generated meshes.
+
+    Args:
+        boundary_points: A list of points that represent the boundary of a face. Here,
+            each point is a list of X, Y, and Z cordinates of the point.
+        holes_points: A list of lists of points that represent the boundary of the holes
+            in the face. Here, each point is a list of X, Y, and Z cordinates of the
+            point. Defaults to None. Which will mean the face has no holes.
+
+    Returns:
+        A list of list of points. Each list has vertices for a triangular polygon.
+        Example of structure will be [[X, Y, Z], [X, Y, Z], [X, Y, Z]].
+    """
+    # Create Ladybug Point3D objects from boundary points
+    lb_boundary_pts = [
+        Point3D(point[0], point[1], point[2])
+        for point in boundary_points]
```

Comment:
It is a style thing but I honestly think keeping it in a single line is easier to read. Also you can use * to unwrap the input arguments.

```suggestion
    lb_boundary_pts = [Point3D(*point) for point in boundary_points]
```